news/_posts/core-weekly/2018-10-12-coreweekly-week-40-2018.md

```diff
@@ -0,0 +1,138 @@
+* [#10857](https://github.com/PrestaShop/PrestaShop/pull/10857): Use parent javascripts used for displaying help, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#10862](https://github.com/PrestaShop/PrestaShop/pull/10862): Fixes bug when filter reset functionality sometimes does not work. Thank you [@tomas862](https://github.com/tomas862)
+
+
+### Front Office
+
+* [#10750](https://github.com/PrestaShop/PrestaShop/pull/10750): Do not display product Reference if empty, by [@khouloudbelguith](https://github.com/khouloudbelguith)
+
+
+### Install
+
+* [#10296](https://github.com/PrestaShop/PrestaShop/pull/10296): Init Kernel when possible during installation process, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+
+
+##  Code changes in the '1.7.1.x' branch (for v1.7.4.3)
+
+### Core
+
+* [#10829](https://github.com/PrestaShop/PrestaShop/pull/10829): Hide arguments in debug trace page, by [@jolelievre](https://github.com/jolelievre)
+
+
+## Code changes in modules, themes & tools
+
+### Contact form
+
+* [#25](https://github.com/PrestaShop/contactform/pull/25): Comply to validator.prestashop.com rules, by [@matks](https://github.com/matks)
+
+
+### Shopping cart
+
+* [#28](https://github.com/PrestaShop/ps_shoppingcart/pull/28): Fix missing call to parent::initContent(), by [@eternoendless](https://github.com/eternoendless)
+
+
+### MBO
+
+* [#51](https://github.com/PrestaShop/ps_mbo/pull/51): Filter catalog page. Thank you [@MrBaiame](https://github.com/MrBaiame)
+* [#52](https://github.com/PrestaShop/ps_mbo/pull/52): Prepare template to override, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+
+### Translation Tools Bundles
+
+* [#48](https://github.com/PrestaShop/TranslationToolsBundle/pull/48): Improve node visitor, by [@eternoendless](https://github.com/eternoendless)
+* [#49](https://github.com/PrestaShop/TranslationToolsBundle/pull/49): Add support for extraction of wordings in form type classes, by [@eternoendless](https://github.com/eternoendless)
+* [#50](https://github.com/PrestaShop/TranslationToolsBundle/pull/50): Always use realpath on root_dir, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#51](https://github.com/PrestaShop/TranslationToolsBundle/pull/51): Use new name for translator service, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+
+### Google Analytics
+
+* [#14](https://github.com/PrestaShop/ps_googleanalytics/pull/14): Only cast as array when a product is returned, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+
+
+### Favicon Notification
+
+* [#6](https://github.com/PrestaShop/ps_faviconnotificationbo/pull/6): Fix issues. Thank you [@clementdaubeuf](https://github.com/clementdaubeuf)
+
+
+<hr />
+
+Thank you to the contributors whose pull requests were merged since the last Core Weekly Report: @clementdaubeuf, @DanielBoettner, @MrBaiame, @Mreker, @sarjon, @SebBareyre, @tomas862!
```

Comment:
```diff
- , @tomas862
+ and @tomas862
```

wdyt?

Reply:
details ?

Reply:
I think it more "french / english" when you finish to list a list (xD) with an "and"